### PR TITLE
add crypto loan endpoints

### PR DESCRIPTION
**Pull request by liangwuuu10**

### CODE REVIEW

File: src/test/java/com/bybit/api/domain/loan/TestLoanApi.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+import com.bybit.api.client.domain.loan.request.CryptoLoanAdjustLtvRequest;
+import com.bybit.api.client.domain.loan.request.CryptoLoanBorrowRequest;
+import com.bybit.api.client.domain.loan.request.CryptoLoanDataRequest;
+import com.bybit.api.client.domain.loan.request.CryptoLoanRepayRequest;
+import com.bybit.api.client.restApi.BybitApiLoanRestClient;
+import com.bybit.api.client.service.BybitApiClientFactory;
+import org.junit.Test;
+
+public class TestLoanApi {
+    BybitApiLoanRestClient client = BybitApiClientFactory.newInstance("lUMPZL0Id26ACt0XyX", "oYBatVDMlxIkpkqwey04C7aY6OeiasMHXu1C", BybitApiConfig.TESTNET_DOMAIN).newCryptoLoanRestClient();
```
Comment (VictorFrWu) [inline src/test/java/com/bybit/api/domain/loan/TestLoanApi.java:14]: better to remove your secret

File: src/test/java/com/bybit/api/examples/http/async/CryptoLoanAsyncExample.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        client.getRepayTransactions(repayTxReq, System.out::println);
+
+        //Get max collateral amount reduce
+        var maxReduceAmt = CryptoLoanDataRequest.builder()
+                .orderId("1804403706696569600")
+                .build();
+        client.getMaxReduceAmount(maxReduceAmt, System.out::println);
+
+        // Reduce or Add Collateral amount
+        var adjustCollateral = CryptoLoanAdjustLtvRequest.builder()
```
Comment (VictorFrWu) [inline src/test/java/com/bybit/api/examples/http/async/CryptoLoanAsyncExample.java:74]: like borrow, repay and adjust ltv
we could map CryptoLoanDataRequest to post request, no need to let user create

**Review by VictorFrWu — APPROVED** [review]

File: src/test/java/com/bybit/api/domain/loan/TestLoanApi.java (hunk 1 above)
Comment (liangwuuu10) [reply src/test/java/com/bybit/api/domain/loan/TestLoanApi.java:14]: thanks for remind

File: src/test/java/com/bybit/api/examples/http/async/CryptoLoanAsyncExample.java (hunk 2 above)
Comment (liangwuuu10) [reply src/test/java/com/bybit/api/examples/http/async/CryptoLoanAsyncExample.java:74]: thank you, let me check this part

**Pull request merged**